src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -80,7 +80,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
-    UUID userID = UserStore.getInstance().getUser(username).getId();
+    UUID userID = userStore.getUser(username).getId();
```

Comment:
The `userStore.getUser(username)` portion here is a repeat of line 76, where you called it to create the variable `user`, so here you can just say `user.getId()`.